jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibPlugin.java

```diff
@@ -87,17 +87,36 @@ public void apply(Project project) {
     Task buildImageTask =
         project
             .getTasks()
-            .create(BUILD_IMAGE_TASK_NAME, BuildImageTask.class)
+            .create(
+                BUILD_IMAGE_TASK_NAME,
+                BuildImageTask.class,
+                task -> {
+                  task.setGroup("Jib Docker");
```

Comment:
Might be better to just go with "Jib" instead of  "Jib Docker". Jib doesn't actually use Docker for most of its execution (only for loading to the daemon in `jibDockerBuild`).

Reply:
Sounds reasonable, [updated](https://github.com/aivantsov/jib/commit/9c2ce6c918c8d8c041a2f8838a7fb05ed7aba982)